eng/Generate-API.ps1

```diff
@@ -0,0 +1,3 @@
+$servicesProj = Resolve-Path "$PSScriptRoot/service.proj"
+
+dotnet msbuild /p:Configuration=Release /restore /t:GenerateAPIListing $servicesProj
```

Comment:
Shouldn't this be "dotnet build ..."?

Reply:
No, `dotnet msbuild` is the right thing here, I'm not trying to build, just invoke the custom target.

Reply:
Sure, but you can pass targets (/t) to build as well. I suppose as long as dependencies are restored in either case, it really doesn't matter.